docs/index.asciidoc

```diff
@@ -149,6 +150,15 @@ Example:
 
 NOTE: It is an error to specify both `dictionary` and `dictionary_path`.
 
+[id="plugins-{type}s-{plugin}-dictionary_file_max_bytes"]
+===== `dictionary_file_max_bytes`
+
+  * Value type is <<number,number>>
+  * Default value is 3145728 (3MB)
```

Comment:
Given that there didn't use to be a limit, and I don't think disabling limit checking it is an option (or is it?), I'm tempted to put a very high number here, like 128MB or even 1GB

Reply:
There is no option to disable the limit check. I put it 128MB